### PR TITLE
do not enable pod presets, they've been removed from 3.7

### PR DESCRIPTION
**Pull request by bparees**
fixes https://github.com/openshift/origin/issues/16313

### CODE REVIEW

**Comment by bparees** [comment]
@spadgett ptal

**Comment by spadgett** [comment]
@bparees Thanks.

Looks fine to me. My only comment is that it would be nice to enable pod_presets still for `--version=v3.6.0`, but I'm not sure how hard that is.

File: examples/sample-app/application-template-stibuild.json [hunk 1]
```diff
@@ -485,6 +485,7 @@
     }
   ],
   "labels": {
-    "template": "application-template-stibuild"
+    "template": "application-template-stibuild",
+    "testlabel": "${MYSQL_USER}"
```
Comment (spadgett) [inline examples/sample-app/application-template-stibuild.json]: Did you mean to add this?

Comment (bparees) [reply examples/sample-app/application-template-stibuild.json]: err, no. :)

will remove.

**Comment by bparees** [comment]
@spadgett removed the extra file.

**Comment by bparees** [comment]
(and people who want the podpreset behavior can still grab a v3.6 oc binary and use that)

File: pkg/oc/bootstrap/docker/openshift/helper.go [hunk 2]
```diff
@@ -848,7 +848,6 @@ func (h *Helper) updateConfig(configDir string, opt *StartOptions) error {
 window.OPENSHIFT_CONSTANTS.ENABLE_TECH_PREVIEW_FEATURE = {
   service_catalog_landing_page: true,
   template_service_broker: true,
```
Comment (spadgett) [inline pkg/oc/bootstrap/docker/openshift/helper.go]: Trailing comma here

**Comment by spadgett** [comment]
> not to mention the hassles of maintaining version-specific behavior in cluster up

We might need version-specific behavior anyway. Eventually this extension should go away entirely in 3.7 since these we will be the defaults. But we can cross that bridge when we come to it.

**Comment by bparees** [comment]
> We might need version-specific behavior anyway. Eventually this extension should go away entirely in 3.7 since these we will be the defaults. But we can cross that bridge when we come to it.

we dropped support for running the SC with a 3.6 cluster, using a 3.7 client binary, already.  There were too many other changes in how it runs in 3.7 vs 3.6.

File: pkg/oc/bootstrap/docker/openshift/helper.go (hunk 2 above)
Comment (bparees) [reply pkg/oc/bootstrap/docker/openshift/helper.go]: yeah i confused myself into thinking that was go code when the tests were failing last night. sigh. fixe.d

**Comment by spadgett** [comment]
OK, thanks @bparees 

/lgtm

**Pull request merged**